catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -201,7 +181,7 @@ public void init() {
   private Object configure() {
     if (StringUtils.isEmpty(monitoredDirectory)) {
       LOGGER.warn("Cannot setup camel route - must specify a directory to be monitored");
-      return null;
+      throw new IllegalArgumentException("Property {monitoredDirectory} cannot be empty");
```

Comment:
Seems like just the exception would be enough, do we need the warning too?

Reply:
Agreed; I don't think we should log AND throw here